### PR TITLE
Color view

### LINKED ISSUE #1288: New project UI: colour picker
- [x] User can access colour picker popup when creating a new project (from 'New project' view).
- [x] Choosing an item from the popup, immediately commits the option and closes the popup. (Applies only for the free colour picker.)
- [x] Paid colour picker: To confirm and close the popup, a user must tap 'Done' button.
- [x] Touching outside of the popup, or pressing the system back button, cancel the action and closes the popup.
- [x] Use the checkbox to indicate selected colour.
- [x]  Show default colours to all users and picker only to users who are under starter, premium and enterprise workspaces.
- [x] When a user uses picker then fill the empty circle with this colour and add a checkbox.

Other notes:
- List of free colours are available here #803

### Free colour picker
new project colour picker

Zeplin: https://zpl.io/be1pk9B

### Paid colour picker
new project colour picker paid

Zeplin: https://zpl.io/V1ed9Em

### PR DESCRIPTION
**Pull request by willsb**
Closes #1288 

Functional test while reviewing.

### CODE REVIEW

File: Toggl.Giskard/Resources/drawable/ValueSliderThumb.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿<shape xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (kornelijepetak) [inline Toggl.Giskard/Resources/drawable/ValueSliderThumb.xml]: What's going on with whitespace in this file?

File: Toggl.Giskard/Views/CircleView.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        {
+        }
+
+        public CircleView(Context context, IAttributeSet attrs, int defStyle)
+            : base(context, attrs, defStyle)
+        {
+            paint = new Paint { Flags = PaintFlags.AntiAlias };
+        }
+
+        protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
```
Comment (kornelijepetak) [inline Toggl.Giskard/Views/CircleView.cs]: 🔥 superfluous

File: Toggl.Giskard/Views/HueSaturationPickerView.cs [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+            var x = (Width * Hue) - circleRadius;
+            var y = Height * complement(Saturation) - circleRadius;
+
+            var circle = new RectF(x, y, x + circleDiameter, y + circleDiameter);
+            canvas.DrawRoundRect(circle, circleRadius, circleRadius, circleStrokePaint);
+            canvas.DrawRoundRect(circle, circleRadius, circleRadius, circleFillPaint);
+        }
+
+        private float complement(float number) => Math.Abs(number - 1);
```
Comment (kornelijepetak) [inline Toggl.Giskard/Views/HueSaturationPickerView.cs]: Can you move this to static using for consistency with the rest of the app?

File: Toggl.Giskard/Views/CircleView.cs (hunk 2 above)
Comment (willsb) [reply Toggl.Giskard/Views/CircleView.cs]: How so? This ensures the view is always square (and that the circle is a circle, not an ellipsis)

Comment (kornelijepetak) [reply Toggl.Giskard/Views/CircleView.cs]: My bad. I haven't read this correctly.

**Pull request merged**